dev/MRTCore/mrt/Core/src/MRM.cpp

```diff
@@ -899,24 +903,38 @@ STDAPI MrmGetFilePathFromName(_In_ PCWSTR filename, _Outptr_ PWSTR* filePath)
         pointerToPath++;
     }
 
+    pointerToPath = pathAllocated ? pathAllocated.get() : path;
+
+    wchar_t moduleFilename[MAX_PATH];
+    RETURN_IF_FAILED(StringCchCopy(moduleFilename, ARRAYSIZE(moduleFilename), lastSlash != nullptr ? lastSlash + 1 : pointerToPath));
```

Comment:
nit: The rest of this function seems to use `StringCchCopyW` directly (instead of `StringCchCopy`).